proposals/distribution.md

```diff
@@ -0,0 +1,61 @@
+
+In the OCI, having a solid, common distribution specification with conformance testing will ensure long lasting security and interoperability throughout the container ecosystem.
+
+## Proposal
+
+TL;DR; Move [https://github.com/docker/distribution/tree/master/docs/spec](https://github.com/docker/distribution/tree/master/docs/spec) to [https://github.com/opencontainers/distribution-spec](https://github.com/opencontainers/distribution-spec)
+
+This proposal covers the distribution API spec, and while it does not cover the code for the docker-registry, that implementation is considered the reference implementation. There are other implementations of this protocol, not all are open-source though (Google gcr.io, Amazon ECR, CoreOS Quay, Gitlab registry, JFrog Artifactory registry, Huawei Dockyard, etc).
+
+In the past when the topic of having an OCI specification around the distribution of container images, it was deferred as "let’s get the image format defined, mean while the industry will settle on a distribution standard". Fast forward, OCI image format is out and adopted, and the Registry v2 is the defacto standard. There is and will be use-cases for alternate methods and the future will likely hold creative ways to push, fetch and share container images, but right now this promotion serves to acknowledge by the OCI the current industry standard of distributing container images.
+
+There is polish that is needed e.g. broken links to storage-driver docs, as well as making sections more generic regarding the OCI descriptors and media-types, but on the whole this is a lateral move.
+
+## Initial Maintainers
+
+* Stephen Day <stephen.day@docker.com> (@stevvooe)
+* Vincent Batts <vbatts@redhat.com> (@vbatts)
+* Derek McGowan <derek.mcgowan@docker.com> (@dmcgowan)
+
+Additional Maintainers to consider:
+
+* Ahmet Alp Balkan (Google)
+* Matt Moore (Google)
+* Yuwa (MSFT)
+* Clayton Coleman (Red Hat)
+* Antonio Murdaca (@runcom) (Red Hat)
+* Samuel Karp (@samuelkarp) (AWS)
+* Mike Brown (IBM)
+* Jimmy Zelinskie jimmy@coreos.com (@jzelinskie)
+* Liu Genping <[liugenping@huawei.com](mailto:liugenping@huawei.com)>
+* Vanessa Sochat (@vsoch)
+
+## Code of Conduct
+
+This project would incorporate (by reference) the OCI Code of Conduct ([https://github.com/opencontainers/tob/blob/master/code-of-conduct.md](https://github.com/opencontainers/tob/blob/master/code-of-conduct.md)).
+
+## Governance and Releases
+
+This project would incorporate the Governance and Releases processes from the OCI project template: [https://github.com/opencontainers/project-template](https://github.com/opencontainers/project-template).
+
+## Project Communications
+
+Both of the proposed projects would continue to use existing channels in use by the OCI developer community for communication including:
+
+* GitHub for issues and pull requests
+* The dev@opencontainers.org email list
+* The monthly OCI developer community conference call
```

Comment:
Does starting at v2 (as suggested below) get us out of the [pre-1.0 weekly meeting][1] recommendation?  I'm not sure if the "we're starting at v2" approach was considered when writing those docs (I certainly hadn't considered it).

[1]: https://github.com/opencontainers/project-template/blame/8e4c7e3dbea403c727d7fc526b8dcd53d6c98202/RELEASES.md#L34

Reply:
It was discussed in the google doc

Reply:
hey @wking here you go! If you follow the link at the top, click on "Comments" in the top right, you can explore some of the discussion.

![image](https://user-images.githubusercontent.com/814322/35313600-22283afc-0076-11e8-9494-e24f6ad8e432.png)

Reply:
I'm fine with starting at v2.  I'm just not clear on whether monthly meetings are the right target level for a new spec (regardless of the number we put on our initial release).  The discussion in this PR suggests folks are going to have lots of ideas.  Project-template suggests weekly meetings during pre-1.0 development (presumably because the roadmap is less obvious/established then).  And there's no reason that roadmap discussions and such couldn't happen asynchronously (but then maybe we want to drop the meeting recommendation from project-template?).  Anyhow, I'd just like to make sure that, if this project starts out with monthly meetings, it was a conscious decision that we didn't expect to need weekly meetings.

Reply:
@wking sounds like something to discuss on the next monthly meeting :-)

Reply:
While there is plenty of discussion here, not such an overhaul needed as to change increase from monthly. I am sure we'll adjust the meeting schedule as needed